api/src/main/java/org/apache/cloudstack/api/BaseCmd.java

```diff
@@ -270,9 +267,22 @@ public String getActualCommandName() {
         } else {
             cmdName = this.getClass().getName();
         }
-       return cmdName;
+        return cmdName;
     }
 
+    public static String getCommandNameByClass(Class<?> clazz) {
```

Comment:
getActualCommandName is not needed then

Reply:
I noticed there was kind of a duplicated code,  I changed

Reply:
Hello, @weizhouapache.

With these new changes, `getActualCommandName` will be a facade to `getCommandNameByClass`, passing `this.getClass` as parameter.